README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
+2)  Compose and deploy your device profile using the Device Profile Modeling tool provided [here](https://www.edgexfoundry.org/tbd). This will ensure the profile is validated and has all required components.
+2) Run your device locally, configured to point at the cloud instance of the deployed EdgeX to send data to either an MQTT or REST device service.
+3) Once deployed, Run the verification script provided to ensure that core-data has been correctly populated with the data that was sent in the previous step.
```

Comment:
So, we went from "Device Profile Modeling tool" in your step 2 to a "script" in this step 3 (technically, 4).

I need to think about what this script would do aside from being a JSON pretty-printer.  It can see the logic in there being a verification step, but, I'm not seeing how we can help here.  Are you imagining, say, an instrumented device service that logs the data it receives and an app services configurable that extracts the presumably same data from Core Data and a script that compares the two?

Reply:
I'm imagining a script that inputs the sample data -- the same data that gets sent to the device service -- and compares it to a core data database query to inspect that the data is all there, properly formatted, and maybe check other things like metadata to ensure that things are registered and described properly.

Reply:
Yes step 2 is about building the profile, step 4 is about ensuring that the data you sent is properly ingested and properly described.